website/docs/docs/build/metricflow-time-spine.md

```diff
@@ -124,8 +166,44 @@ final as (
 
 select *
 from final
+-- filter the time spine to a specific range
+where date_day > dateadd(year, -4, current_timestamp()) 
+and date_hour < dateadd(day, 30, current_timestamp())
 ```
 
 </VersionBlock>
 
-You only need to include the `date_day` column in the table. MetricFlow can handle broader levels of detail, but it doesn't currently support finer grains.
```

Comment:
@mirnawong1 for old versions, we can update this to say: 
"...but finer grains are only supported in versions 1.9+."